Feature/physics docs (#407)
ai2cm/pace: docs/physics/state.rst

> .. _state:

Reviewer: This tag will conflict with `docs/util/state.rst`, please change the label.

Reviewer: What was done to resolve this?

Author: removed

=====
State
=====

Containers

> ------------

Reviewer: ```suggestion
----------
```

Variables used in physics are packedged using a container type called :py:class:`pace.physics.PhysicsState`.
This contains variables copied from the dynamical core for calculating physics tendencies.

> It also contains sub-container for the individual physics schemes. Currently, it only contains :py:class:`pace.physics.MicrophysicsState`.

Reviewer: ```suggestion
It also contains sub-container for the individual physics schemes.]
Currently, its only sub-container is :py:class:`pace.physics.MicrophysicsState`.
```

> An example to initialize a PhysicsState and MicrophysicsState is shown below:

Reviewer: ```suggestion
You can initialize a zero-filled PhysicsState and MicrophysicsState from other Pace objects as follows:
```
